server/data/data-cassandra/src/main/java/org/apache/james/rrt/cassandra/CassandraMappingsSourcesDAO.java

```diff
@@ -98,4 +105,13 @@ private PreparedStatement prepareRetrieveSourcesStatement(Session session) {
             .flatMapMany(Flux::fromIterable)
             .map(row -> MappingSource.parse(row.getString(SOURCE)));
     }
+
+    /*
+     * Removes all data in the `mappings_sources` projection table
```

Comment:
could you use `{@value full.qualified.domain.name.ClassName#CONSTANT}` to reference to the table name?